.github/workflows/ci.yml

```diff
@@ -0,0 +1,69 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request: ~
+
+jobs:
+  # derived from https://dev.to/felipetofoli/github-actions-for-net-full-framework-build-and-test-299h
+  NET_Tests:
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        dotnet-version: [ '35', '40' ]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup variables
+        id: test_file
+        run: echo "::set-output name=test_file::EasyPost.Tests.Net${{ matrix.dotnet-version }}"
+      - name: Setup MSBuild
+        if: always()
+        uses: microsoft/setup-msbuild@v1.1
+      - name: Setup Nuget
+        uses: NuGet/setup-nuget@v1.0.5
+      - name: Setup VSTest
+        uses: darenm/Setup-VSTest@v1
+      - name: Navigate to project root
+        run: cd $GITHUB_WORKSPACE
+      - name: Restore NuGet Packages
+        run: nuget restore EasyPost.sln
+      - name: Build Solution
+        run: msbuild ${{ steps.test_file.outputs.test_file }}\${{ steps.test_file.outputs.test_file }}.csproj /p:platform="Any CPU" /p:configuration="Test" /p:outputPath="bin/Test" /p:target="Rebuild"
+      - name: Run Tests
+        run: vstest.console.exe ${{ steps.test_file.outputs.test_file }}\bin\Test\${{ steps.test_file.outputs.test_file }}.dll
+        # delete the current DLL to guarantee that the next test will generate and use a new one
```

Comment:
Thanks for this inline comment, helps clarify `why` we are doing what we are doing here.

As for using the `JesseTG/rm` action, is there a reason we are pulling in a dependency to effectively do the following?

```
rm -rf ${{ steps.test_file.outputs.test_file }}\bin\Test\${{ steps.test_file.outputs.test_file
```

If we can simply call the `rm` command, I'd rather do that then depending on an entire dependency to do it for us.

Reply:
We actually don't need to remove the file at all anymore, since these all run on different runners.